ui/src/types.ts

```diff
@@ -24,6 +24,8 @@ export interface Source extends HydraResource {
   id: ResourceId;
   name: string;
   columns: Column[];
+  projectId: ResourceId;
+  actions: Record<string, any>;
```

Comment:
Does it have yo be typed as `any`?

Reply:
Now that I started putting HydraResource in my types, I guess I should also type IOperation. Good catch.

Reply:
I added the proper types in #152. We're gonna have some conflicts after merging this one, but that's fine.